Libs/Image/Image.h

```diff
@@ -13,6 +17,13 @@ class Image
   static const unsigned dims = 3;
   using PixelType = float;
   using ImageType = itk::Image<PixelType, dims>;
+  typedef double CoordinateRepType;
```

Comment:
All of these should be only in the ImageUtils function itself, not in the outer namespace.